third_party/2/google/protobuf/any_pb2.pyi

```diff
@@ -0,0 +1,25 @@
+# @generated by generate_proto_mypy_stubs.py.  Do not edit!
+from typing import (
+    Text,
+)
+
+from google.protobuf.message import (
+    Message,
+)
+
+
+class Any(Message):
+    type_url = ...  # type: Text
+    value = ...  # type: bytes
+
+    def __init__(self,
+                 type_url: Text = ...,
+                 value: bytes = ...,
+                 ) -> None: ...
+
+    @classmethod
+    def FromString(cls, s: bytes) -> Any: ...
+
+    def MergeFrom(self, other_msg: Message) -> None: ...
```

Comment:
MergeFrom/CopyFrom exists on Message now and they are not generic, so these overrides are useless. let's snip them? (i will update the generator with this and the `...` thing)

Reply:
Are those the only ones which should be removed?  Or should I just wait for an updated mypy-protobuf, rather than attempting to find all of the cases by hand?

Reply:
yeah it's just those 2 but it's all the message types so might be a bit annoying to do by hand.